scripts/allocate_ip.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -eu
+
+#Specify full node name as first parameter ~ d52-54-77-77-01-01.virtual.cloud.suse.de
+NODE_NAME="$1"
+
+#Specify network as second parameter
+NETWORK_TO_ALLOCATE="$2"
+
+#Specify range as third parameter
+RANGE_TO_ALLOCATE="$3"
+
+TEMPSCRIPT=$(mktemp)
+
+function remove_tempfile() {
+    rm -f $TEMPSCRIPT
+}
```

Comment:
this function is not called - should be in a 'trap' ?

Reply:
@matthewoliver Please add something like:
```
trap remove_tempfile EXIT
```

Reply:
done